src/internal/api/auth/token.go

```diff
@@ -12,7 +12,8 @@ import (
 func RequireSecret(validToken string) func(http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-			if r.Header.Get("Authorization") != validToken {
+
+			if r.Header.Get("Authorization") != validToken && r.URL.Query().Get("auth") != validToken {
```

Comment:
This is a requirement for default ssr events, I do agree the implementation should be looked into and under less time restrictive circumstances, I also think the auth is pretty weak in general if the expectation is to open this and leave it deployed somewhere. especially since the token is just stored in session.

Reply:
I may just bring in the Microsoft fetch event source to solve this, I'll play with it and make sure it works but that should fix this and move back to header only

Reply:
I ended up removing this and moving forward with @microsoft/fetch-event-source to resolve the need for the query param.